pkg/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -192,13 +192,13 @@ var _ = Describe("Tuned", func() {
 					reservedSet, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
 					Expect(err).ToNot(HaveOccurred())
 					reserveCPUcount := reservedSet.Size()
-					channelsRegex := regexp.MustCompile(`\s*\[net_1\]\\ntype=net\\ndevices_udev_regex=` + devicesUdevRegex + `\\nchannels=combined ` + strconv.Itoa(reserveCPUcount) + `\s*`)
+					channelsRegex := regexp.MustCompile(`\s*\[net\]\\ntype=net\\ndevices_udev_regex=` + devicesUdevRegex + `\\nchannels=combined ` + strconv.Itoa(reserveCPUcount) + `\s*`)
```

Comment:
What was that /s/net_1/net ? a typo?

Reply:
i changed the way I generate the [net] entries inorder to avoid tuned priority issues , so it will always be:
[net], [net_1],.....[net_n]   (n = device entries in PAO, when no device is listed then we have a default [net] )

Reply:
ack